test/service/test_jobs.py

```diff
@@ -226,7 +226,7 @@ def test_sends_correct_payload_to_piazza_pzsvc_ndwi_py(self):
         }, json.loads(self.mock_execute.call_args[0][2]['body']['content']))
 
     def test_does_not_create_database_record_if_cannot_start(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.